document/Supplementary-Note.md

```diff
@@ -0,0 +1,232 @@
+#Ten Simple Rules for taking advantage of GitHub
+
+Yasset Perez-Riverol (1)[^1], Rui Wang (1), Timo Sachsenberg (2), Julian Uszkoreit (3), Laurent Gatto (4), Felipe da Veiga Leprevost (5), Christian Fufezan (6), Tobias Ternent (1), Juan Antonio Vizcaíno (1)[^2]
+
+(1) European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK.
+
+(2) Applied Bioinformatics and Department of Computer Science, University of Tübingen, D-72074 Tübingen, Germany.
+
+(3) Medizinisches Proteom-Center, Ruhr-Universität Bochum, Universitätsstr. 150, D-44801 Bochum, Germany.
+
+(4) Computational Proteomics Unit, Cambridge Systems Biology Centre, University of Cambridge Tennis Court Road Cambridge, CB2 1GA, UK.
+
+(5) Department of Pathology, University of Michigan, Ann Arbor, Michigan 48109, USA.
+
+(6) Institute of Plant Biology and Biotechnology, University of Muenster, Schlossplatz 8, 48143 Muenster, Germany.
+
+[^1]: yperez@ebi.ac.uk.
+[^2]: juan@ebi.ac.uk.
+
+
+## Section 1: Git LSF file system
+
+Git Large File Storage (LFS) replaces large files (> 50 Mb)such as audio samples, videos, datasets, and graphics with text pointers inside Git, while
+storing the file contents. In order to use the git _lfs_ service for big files, the user can follows the next steps: 
+
+
+ 1- Download the git plugin from here https://git-lfs.github.com/ or using Homebrew
+ 
+```bash
+
+   brew install git-lfs
+
+```
+  
+ 2- Select the file types you'd like Git LFS to manage (or directly edit your .gitattributes). You can configure additional file extensions at anytime.
+
+```bash
+ 
+   git lfs track "*.psd"
+
+```   
+
+3 -There is no step three. Just commit and push to GitHub as you normally would.
+
+```bash
+ 
+  git add file.psd
+  
+  git commit -m "Add design file"
+  
+  git push origin master
+ 
+``` 
+
+
+One of the major concerns about the use of GitHub is that some of the services are limited and can be charged if the users exceed certain quotes. For example, the Git LSF filesystem is available for all type of projects: personal repositories or organization accounts. A minimal space is provided for personal/organization repositories without charge
+(1GB). If you exceed this quota, you can still clone repositories with large assets, but you will only retrieve the pointer files, and you will not be able to push new files back up.
+ 
```

Comment:
Note that the use of GitHub LFS service is not mandatory and alternative providers, including self-hosting of files is possible.